packages/thumbprint-react/components/Avatar/index.jsx

```diff
@@ -33,6 +34,32 @@ const getBadgeProps = ({ size, hasUnreadNotifications, isChecked, isOnline }) =>
 const shouldShowBadge = ({ size, hasUnreadNotifications, isChecked, isOnline }) =>
     size !== 'xsmall' && (hasUnreadNotifications || isChecked || isOnline);
 
+const BACKGROUND_COLOURS = [
+    tokens.tpColorIndigo600,
+    tokens.tpColorGreen600,
+    tokens.tpColorYellow600,
+    tokens.tpColorRed600,
+    tokens.tpColorPurple600,
+    tokens.tpColorBlue600,
+];
+
+const TEXT_COLOURS = [
```

Comment:
You got the text and background colors mixed up. Should be dark text on a light background.

***

Also, the `BACKGROUND_COLOURS` and `TEXT_COLOURS` would be easier to maintain if they were combined into a single `COLOURS` variable that is an array of objects like `{ text: '#fff', background: '#000' }`. This makes the relationship between the background and text color more explicit.

Reply:
Ah yeah, my bad. Combined the arrays and flipped the fg/bg